src/components/App/EpisodesList/Show/Details/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import styles from './Details.module.scss';
+
+const Details = props => {
+  if (!props.summary) return null;
+
+  return (
+    <div className={`${styles.detailsWrapper} ${styles.visible}`}>
+      <button
+        className={styles.toggleDetailsButton}
+        onClick={props.toggleDetails}
+      >
+        {!props.detailsShown ? 'Show details' : 'Hide details'}
+      </button>
+      <p className={`${props.detailsShown ? styles.visible : ''} ${styles.summary}`}>
+        {props.summary ? props.summary.replace('<p>','').replace('</p>','') : ''}
```

Comment:
I don't understand this `<p>` tag replacing. Won't it be easier to render conditionally Details button in the parent component? Or consider toggle details on click on the poster and add button for the Show link?